src/language-js/print/assignment.js

```diff
@@ -345,6 +345,14 @@ function isPoorlyBreakableMemberOrCallChain(
       return false;
     }
 
+    const hasComplexTypeParams =
+      node.typeParameters &&
+      isNonEmptyArray(node.typeParameters.params) &&
+      willBreak(print("typeParameters"));
```

Comment:
Please add one more condition: if there is more than one type parameter, the call shouldn't be considered "poorly breakable".

Reply:
And another case: the only type parameter is a union type.